src/pyos_sphinx_theme/__init__.py

```diff
@@ -8,7 +8,7 @@
 # from sphinx_book_theme import hash_assets_for_files
 from sphinx.util import logging
 
-__version__ = "0.1.dev0"
+__version__ = "0.3"
```

Comment:
I updated this manually and the distribution files look correct. but ideally we could use that magic _version.py file and import that here to create the` __version__` attribute.